ble_controller/README.rst

```diff
@@ -28,6 +28,7 @@ Each library is distributed in soft-float, softfp-float, and hard-float builds.
   * The libraries are for evaluation purposes only.
   * The libraries are not fully functional.
   * The libraries are not built for performance.
+  * Certain toolchains are known to segfault when linking with the libraries.
```

Comment:
```suggestion
  * Some toolchains cause a segmentation fault when linking the libraries.
```

Reply:
Not sure about "linking the libraries" - "linking in the libraries" or "linking to the libraries" might be better, depending on what you want to say.

Reply:
good catch, "linking with the libraries" does indeed communicate the wrong thing.